src/views/home/home.view.tsx

```diff
@@ -17,12 +24,37 @@ export const Home = (): JSX.Element => {
   const navigate = useNavigate();
   const { formData, setFormData } = useFormContext();
   const { connectedProvider } = useProvidersContext();
+  const [depositLimitModal, setDepositLimitModal] = useState<ModalState<DepositLimitModalData>>({
+    status: "closed",
+  });
 
-  const onFormSubmit = (formData: FormData) => {
+  const onSubmitForm = (formData: FormData) => {
     setFormData(formData);
     navigate(routes.bridgeConfirmation.path);
   };
 
+  const onCheckDepositLimitAndSubmitForm = (formData: FormData) => {
+    if (formData.from.key === "ethereum") {
+      const { from, token } = formData;
+      const depositLimits = DEPOSIT_LIMITS[from.chainId][token.address];
```

Comment:
This is `undefined` for any token not present in `DEPOSIT_LIMITS` and raises runtime errors below.

<img width="639" alt="image" src="https://user-images.githubusercontent.com/15896177/225731910-d2631416-18ff-442c-bf77-f9ce7d471665.png">

Reply:
Right! Good catch, let me fix it.